Homework_8/dictionary.py

```diff
@@ -0,0 +1,58 @@
+"""
+Задача 1: Є система, де є списки користувачів.
+roles = { 'admin' : [...], 'maintainer' : [...], 'manager' : [...], 'developer': [...]}
+В цих списках треба понаписувати імена користувачів.
+При введенні імені необхідно перевірити, до якої ролі належить ця людина та вивести повідомлення виду: "Hello, <role>"
+Якщо імені немає у списках, вивести: "Hello, Guest"
+"""
+
+roles = {'admin': ['Sergey', 'Ethan'], 'manager': ['Anna', 'Helen'], 'developer': ['Sasha', 'Den']}
+user_name = input('Print your name here: ')
```

Comment:
I suppose can be better -> user_name = input('Print your name here: ').title().

Reply:
agree, done